install.sh

```diff
@@ -20,6 +20,7 @@ source install/check-latest-commit.sh
 source install/check-minimum-requirements.sh
 
 # Let's go! Start impacting things.
+source install/upgrade-clickhouse.sh
```

Comment:
The reason this is placed before turn-things-off is because turn things off script will remove the clickhouse image, which we need to verify the version of clickhouse before proceeding

Reply:
Let's add this as a comment in the code, since whoever edits this in the future will probably want to know this fact as well.